gofn.go

```diff
@@ -2,12 +2,27 @@ package gofn
 
 import (
 	docker "github.com/fsouza/go-dockerclient"
+	"github.com/nuveo/gofn/iaas"
 	"github.com/nuveo/gofn/provision"
 )
 
+const dockerPort = ":2375"
+
 // Run runs the designed image
 func Run(buildOpts *provision.BuildOptions, volumeOpts *provision.VolumeOptions) (stdout string, err error) {
 	client := provision.FnClient("")
+	if buildOpts.Iaas != nil {
+		var machine *iaas.Machine
```

Comment:
to avoid error shadowed https://groups.google.com/forum/#!topic/golang-nuts/VeJTB9dhhK4